backend/util.py

```diff
@@ -29,9 +49,10 @@ def __init__(self, code: int, message: str | None = None) -> None:
                 HTTP status code.
             message:
                 The response message to be wrapped into the message dict.
-                Default to "OK" if code is lower than 400, to "Failed" otherwise.
+                Default to "OK" if code is lower than 400, otherwise, return the failed message.
```

Comment:
Default to "OK" if code is lower than 400, otherwise, a corresponding failed message.

Reply:
Fix it on [b3c0026](https://github.com/ntut-xuan/FastShop/pull/55/commits/b3c002651d12f6c45b064c456a01472d1ef0dd4b).